yaspin/core.py

```diff
@@ -170,6 +176,8 @@ def color(self):
     def color(self, value):
         self._color = self._set_color(value) if value else value
         self._color_func = self._compose_color_func()  # update
+        if self._color_func is None:
+            self._warn_color_disabled()
```

Comment:
There is also a way to set color during spinner instantiation (`color` argument in `__init__`), which also seems to be a logical place to throw a warning. I'm thinking that it might be a better idea to move `self._warn_color_disabled()` under `_set_color()`. Same applies for `on_color` and `attrs`.